.github/workflows/elixir_test.yml

```diff
@@ -62,3 +62,14 @@ jobs:
 
       - name: Run dialyzer
         run: mix dialyzer
+
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846
+
+      - name: Run Tests in Docker
+        run: bin/run-tests-in-docker.sh
```

Comment:
Should both the step name and the bin filename mention that those are smoke tests only, not all tests?